org-brain.el

```diff
@@ -535,6 +535,8 @@ EDGE determines if `org-brain-edge-annotation-face-template' should be used."
 
 (defvar org-brain-selected nil "List of selected org-brain entries.")
 
+(defvar org-brain-show-full-entry nil "Show the entire entry contents.")
```

Comment:
Maybe this should be a `defcustom` instead?

Reply:
Done.